docs/reference/builder.md

```diff
@@ -519,14 +519,14 @@ command.
 
 In the *shell* form you can use a `\` (backslash) to continue a single
 RUN instruction onto the next line. For example, consider these two lines:
-```
-RUN /bin/bash -c 'source $HOME/.bashrc ;\
-echo $HOME'
-```
+
+    RUN /bin/bash -c 'source $HOME/.bashrc; \
```

Comment:
Is there a reason you changed this (and removed the "fences")? We're using GitHub flavored markdown https://guides.github.com/features/mastering-markdown/

To improve this, you could add a `Dockerfile` code hint to improve syntax highlighting;


    ```Dockerfile
    RUN /bin/bash -c 'source $HOME/.bashrc ;\
    echo $HOME'
     ```